### PR TITLE
Update textarea widget documentation: Formatting and content improvements (issue: #10900)

### PR DESCRIPTION
**Pull request by mihir2004**
### Pull Request Description
This pull request implements necessary updates to the textarea widget documentation as outlined in the assigned issue (#10900). The changes include:

- **Formatting Updates:**
  - Replaced all instances of "Fx" with "fx" to ensure consistency throughout the documentation.
  - Adjusted the CSS for all h2 headers to remove the `24px` padding-bottom, retaining only `24px` padding-top for a cleaner layout.

- **Content Updates:**
  - Changed the header "Layout" to "Devices" in the layout table for better clarity.
  - Updated the Description and Expected Value columns to provide more accurate information regarding the textarea widget.

### CODE REVIEW

File: .gitignore
Comment (RudraPratik30) [inline .gitignore:1]: Hey @mihir2004,
You were expected to edit only two files that are mentioned in the issue and rest of the files should be kept untouched.
Please revert all the changes from this file.

File: package-lock.json
Comment (RudraPratik30) [inline package-lock.json:1]: Please revert the changes from this file.

File: package.json
Comment (RudraPratik30) [inline package.json:1]: Please revert the changes from this file.

File: plugins/package-lock.json
Comment (RudraPratik30) [inline plugins/package-lock.json:1]: Please revert the changes from this file.

File: plugins/package.json
Comment (RudraPratik30) [inline plugins/package.json:1]: Please revert the changes from this file.

File: docs/versioned_docs/version-2.50.0-LTS/widgets/textarea.md [hunk 1]
```diff
@@ -52,37 +53,37 @@ In the input field under **Tooltip**, you can enter some text and the component
...
+| Visibility                                  | Controls the visibility of the component. If set to `{{false}}`, the component will not be visible after the app is deployed. | Use the toggle button OR click on **fx** to pass a boolean value or a logical expression that returns a boolean value i.e. either `{{true}}` or `{{false}}` |
+| Disable                                     | Makes the component non-functional when set to true.                                                                          | Use the toggle button OR click on **fx** to pass a boolean value or a logical expression that returns a boolean value i.e. either `{{true}}` or `{{false}}` |
+| Border radius                               | Adjusts the roundness of the component's corners.                                                                             | Numeric value                                                                                                                                               |
 
 </div>
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px''}}>
```
Comment (RudraPratik30) [inline docs/versioned_docs/version-2.50.0-LTS/widgets/textarea.md:81]: Hey @mihir2004, I can see there are some syntax error in this line, can you please correct it?

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @mihir2004, please make the mentioned changes in both versions.

**Comment by mihir2004** [comment]
@RudraPratik30 I have made the necessary changes.

File: docs/docs/widgets/textarea.md [hunk 2]
```diff
@@ -52,37 +53,37 @@ In the input field under **Tooltip**, you can enter some text and the component
...
-|:--------------- |:----------------------------------------- | :------------------------------------------------------------------------------------------------------------- |
-| Show on desktop | Toggle on or off to display desktop view. | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
-| Show on mobile  | Toggle on or off to display mobile view.  | You can programmatically determining the value by clicking on `Fx` to set the value `{{true}}` or `{{false}}` |
+| <div style={{ width:"100px"}}> Property </div> | <div style={{ width:"100px"}}> Description </div> | <div style={{ width:"135px"}}> Expected Value </div>                                                            |
+| :--------------------------------------------- | :------------------------------------------------ | :-------------------------------------------------------------------------------------------------------------- |
+| Show on desktop                                | Toggle on or off to display desktop view.         | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/textarea.md:62]: Hey @mihir2004 you need to update the content of expected value column from the reference given to you in issue.

File: docs/docs/widgets/textarea.md [hunk 3]
```diff
@@ -52,37 +53,37 @@ In the input field under **Tooltip**, you can enter some text and the component
...
+| Show on mobile                                 | Toggle on or off to display mobile view.          | You can programmatically determining the value by clicking on **fx** to set the value `{{true}}` or `{{false}}` |
```
Comment (RudraPratik30) [inline docs/docs/widgets/textarea.md:63]: Please also add the full stop wherever necessary

File: docs/docs/widgets/textarea.md [hunk 4]
```diff
@@ -2,45 +2,46 @@
...
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Exposed Variables
 
-| Variables | Description | How To Access |
-|:---------:|:-----------:|:-------------:|
+|                  Variables                  |                            Description                            |                                    How To Access                                     |
+| :-----------------------------------------: | :---------------------------------------------------------------: | :----------------------------------------------------------------------------------: |
```
Comment (RudraPratik30) [inline docs/docs/widgets/textarea.md:39]: Please make this table left aligned.

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey @mihir2004 please make the mentioned changes in both versions, thanks!

File: docs/docs/widgets/textarea.md (hunk 2 above)
Comment (mihir2004) [reply docs/docs/widgets/textarea.md:62]: Hey I did not understand this. Do I need to replace to content and put In that of text input ?.

File: docs/docs/widgets/textarea.md (hunk 3 above)
Comment (mihir2004) [reply docs/docs/widgets/textarea.md:63]: Got it

File: docs/docs/widgets/textarea.md (hunk 2 above)
Comment (RudraPratik30) [reply docs/docs/widgets/textarea.md:62]: Hey, please update the content of the expected value column exactly as given in the text input docs, i.e. instead of ` You can programmatically determining the value by clicking on **fx** to set the value {{true}} or {{false}} ` we need `You can set it with the toggle button or dynamically configure the value by clicking on fx and entering a logical expression.` there.
let me know if you need any further assistance, thanks!

**Comment by Advitiya-IITRPR** [comment]
@RudraPratik30  I have made the necessary changes, please verify

File: docs/docs/widgets/textarea.md [hunk 5]
```diff
@@ -2,45 +2,47 @@
...
+<div style={{paddingTop:'24px'}}>
 
 ## Exposed Variables
 
-| Variables | Description | How To Access |
-|:---------:|:-----------:|:-------------:|
-| <div style={{ width:"100px"}}> value </div> | This variable holds the value entered in the text area component. | Access the value dynamically using JS. For example, `{{components.textarea1.value}}` |
+| Variable                                                          | Description                                                                          | How To Access |
+| :---------------------------------------------------------------- | :----------------------------------------------------------------------------------- | :------------ |
+| <div style={{ width:"100px"}}> value </div>                       |
```
Comment (RudraPratik30) [inline docs/docs/widgets/textarea.md:40]: Hey, no changes were expected in this table but still, some changes are made and the table is breaking because of that, can you please correct it?

image

**Review by RudraPratik30 — CHANGES_REQUESTED** [review]
Hey, can you please update the mentioned change in both versions? Also can you please add full stops wherever necessary.

**Comment by PriteshKiri** [comment]
Closing the PR due to lack of progress
